hhs_oauth_server/request_logging.py

```diff
@@ -315,6 +325,19 @@ def __str__(self):
         elif getattr(self.response, 'content', False):
             self.log_msg['size'] = len(self.response.content)
 
+        self.log_msg['x-ratelimit-limit'] = self.response.get('x-ratelimit-limit', None)
+        self.log_msg['x-ratelimit-remaining'] = self.response.get('x-ratelimit-remaining', None)
```

Comment:
Since rate limiting is now done at the Akamai level, I'm not sure how actionable this is to log?  It could be very out of sync with whatever rate limiting is happening in Akamai.

Reply:
> Since rate limiting is now done at the Akamai level, I'm not sure how actionable this is to log? It could be very out of sync with whatever rate limiting is happening in Akamai.

@njdister TY for pointing this out!   I just removed those two items.